### PR TITLE
Draft Readme

### PR DESCRIPTION
**Pull request by lzdanski**
Draft of content. Specific areas needed for review:
* Technical details (cloudant noSQL, express framework). I think the general location might work, but I need more info.
* Include the steps minus some of the detail? Keep the full steps from the original readme? No steps at all?

Thanks!

### CODE REVIEW

**Comment by opiethehokie** [comment]
For express it could link to https://expressjs.com/. For Cloudant it could link to https://console.bluemix.net/catalog/services/cloudant-nosql-db. And it could probably link to the https://www.npmjs.com/package/cloudant library for how the app connects to Cloudant.

I think after linking to the detailed instructions, all the steps after that could be removed.

**Comment by lzdanski** [comment]
@opiethehokie, added hyperlinks and removed extra steps. I think leaving the "Before you begin" section would allow more familiar users to just pick up right there without having to go to the tutorial. Thoughts?

**Comment by opiethehokie** [comment]
OK I'm fine with keeping the "Before you begin" there, that makes sense.

File: README.md [hunk 1]
```diff
@@ -1,133 +1,15 @@
 
-# Getting started with Node.js on Bluemix
-By following this guide, you'll set up a development environment, deploy an app locally and on Bluemix, and integrate a Bluemix database service in your app.
+# Node.js getting started application
+The Bluemix Getting started tutorial for Node.js uses this sample application to provide you with a sample workflow for working with any Node.js app on Bluemix; you set up a development environment, deploy an app locally and on Bluemix, and integrate a Bluemix database service in your app.
+
+The Node.JS app uses [Express Framework](https://expressjs.com) and [Cloudant noSQL DB service](https://console.bluemix.net/catalog/services/cloudant-nosql-db) to add information to a database and then return information from a database to the UI. To learn more about how the app connects to Cloudant, see the [Cloudant library for Node.js](https://www.npmjs.com/package/cloudant).
```
Comment (opiethehokie) [inline README.md]: I think this should be Node.js (lowercase js).

**Comment by lzdanski** [comment]
@opiethehokie Typo fixed. Everything looks good on my end, so once you're ready to approve, it's ready to be merged.

**Pull request merged**